lib/plugins/aws/lib/getServiceState.js

```diff
@@ -0,0 +1,12 @@
+const path = require('path');
+
+module.exports = {
+  getServiceState() {
```

Comment:
Sticked to this export schema so that we can re-use this easily in existing plugins (and therefore use `this.serverless...`).